index.css

```
@@ -0,0 +1,86 @@
+body{
+    font-family: "Inter", sans-serif;
+    font-size: 20px;
+    padding: 0;
+    margin: 0;
+}
+h1{
+    font-size: 80px;
+    font-weight: 500;
+}
+h2{
+    font-size: 64px;
+    font-weight: 500;
+}
+h3{
+    font-size: 48px;
+    color: #fff;
+    text-align: center;
+    font-weight: 500;
+}
+h4{
+    font-size: 36px;
+    font-weight: 500;
+}
+.container{
+    width: 1380px;
+    margin: 0 auto; /*позиционирование по центру*/
+}
```

Пока мы эту тему не проходили, тоже на будущее скорее.
```suggestion
.container{
    max-width: 1380px;
    width: 100%;
    margin: 0 auto; /*позиционирование по центру*/
}
```
Это довольно стандартный приём для создания "резиной" вёрстки. При уменьшении экрана ширина контейнера будет адаптироваться)